cli/cli_progress.go

```diff
@@ -20,13 +20,13 @@ const (
 )
 
 type progressFlags struct {
-	enableProgress         bool
+	disableProgress        bool
 	progressUpdateInterval time.Duration
 	out                    textOutput
 }
 
 func (p *progressFlags) setup(svc appServices, app *kingpin.Application) {
-	app.Flag("progress", "Enable progress bar").Hidden().Default("true").BoolVar(&p.enableProgress)
+	app.Flag("disable-progress", "Disable progress bar").BoolVar(&p.disableProgress)
```

Comment:
I don't think we should remove existing flag. Also `--no-progress` is automatically added since this is a boolean flag, so perhaps all we need is better documentation, like this:

```
app.Flag("progress", "Enable progress bar (--no-progress to disable)").Default("true").BoolVar(&p.enableProgress)
```

Reply:
To me it doesn't make sense to have `--progress`. Why would I ever specify `--progress`? It defaults to true.

Reply:
Thats correct. But it also adds no-progress which is what folks use today.

Reply:
Also, your change will add `--no-disable-progress` :)

So how about we keep `progress` as-is for backwards compatibility (and hidden) and add this new flag which is arguably much more discoverable. Progress will be disabled if `disableProgress==true || progress == false`